cloud-tasks/app/app.yaml

```diff
@@ -0,0 +1,23 @@
+ # Copyright 2019 Google LLC
```

Comment:
Include the standard Apache Header as specified by OSPO here and in other files that can include comments.

http://go/releasing/preparing#Apache-header

Reply:
ack.